File: Web/RMA-Web-Shell/projects/claimcare/src/app/claim-manager/shared/entities/tracer-model.ts
import { RolePlayer } from 'projects/clientcare/src/app/policy-manager/shared/entities/roleplayer';
import { RolePlayerBankingDetail } from 'projects/shared-components-lib/src/lib/models/banking-details.model';

export class TracerModel {
  claimId: number;
  rolePlayer: RolePlayer;
  totalAmountPaid: number;
  funeralTracingMaxAmount: number;

  rolePlayerBankingDetails: RolePlayerBankingDetail;
}


export class ClaimTracerPaymentModel {
  tracerInvoiceId: number;
  claimId: number;
  rolePlayerId: number;
  tracingFee: number;
  paymentStatusId: number;
  reason: string;
  payDate: any;
  claimInvoiceType: number;
  invoiceDate: Date;
  capturedDate: Date;
  productId: number;
  bankAccountId: number;
  tracerEmail: string;
}

File: Service/ClaimCare/RMA.Service.ClaimCare.Contracts/Entities/TracerModel.cs
using RMA.Service.ClientCare.Contracts.Entities.RolePlayer;

namespace RMA.Service.ClaimCare.Contracts.Entities
{
    public class TracerModel
    {
        public int ClaimId { get; set; }
        public RolePlayer RolePlayer { get; set; }
        public decimal TotalAmountPaid { get; set; }
        public decimal FuneralTracingMaxAmount { get; set; }
    }
}
